src/main/java/com/basesyntax/dao/OperationHandler.java

```diff
@@ -0,0 +1,7 @@
+package com.basesyntax.dao;
+
+import com.basesyntax.model.Fruit;
+
+public interface OperationHandler {
```

Comment:
OperationHandler is not dao layer class, it belongs to the strategy pattern and contains a business logic. But Dao service must contain only simple crud methods and have a name in the next format `<the entity you work with>Dao. For example, FruitDao